decepticonlp/metrics/temporal_metrics.py

```diff
@@ -0,0 +1,55 @@
+import torch
+import torch.nn as nn
+
+
+class RankCharacters:
```

Comment:
Is it ranking characters or words?

Reply:
My bad, this is rank characters. Before this I did it on a character level but later changed it to a word level, forgot to change the class name :P. I'll change this rn.